packages/base/src/commands.ts

```diff
@@ -17,7 +17,8 @@ import { ITranslator } from '@jupyterlab/translation';
 import { CommandRegistry } from '@lumino/commands';
 import { ReadonlyPartialJSONObject } from '@lumino/coreutils';
 import { CommandIDs, icons } from './constants';
-import { CreationFormDialog } from './dialogs/formdialog';
+// @ts-ignore
```

Comment:
Why do we need a ts-ignore? Maybe a leftover?

```suggestion
```

Reply:
Ahh, for some reason - the next line is showing a silent error locally which was not affecting the build. But no more now, i'll merge this suggestion.